agents/cyberpower_ssh/fence_cyberpower_ssh.py

```diff
@@ -0,0 +1,71 @@
+#!@PYTHON@ -tt
+
+#####
+##
+## Fence agent for CyberPower based SSH-capable power strip
+## Tested with CyberPower model PDU41001, ePDU Firmware version 1.2.0
+##
+#####
+
+import sys, re, time
+import atexit
+sys.path.append("@FENCEAGENTSLIBDIR@")
+from fencing import *
+from fencing import fail, EC_STATUS
+
+def set_power_status(conn, options):
+
```

Comment:
You should remove this line.

Reply:
Not the import line... the empty line at the beginning of the set_power_status().

Reply:
Next commit coming soon...

Reply:
You'll need to readd the import-line, or it'll fail.

Reply:
The fail and EC_STATUS items aren't imported by the "from fencing import *" line?

Reply:
No. * imports a pre-defined set of the fencing library.